types/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for unist-util-inspect 4.1
+// Project: https://github.com/syntax-tree/unist-util-inspect/#readme
+// Definitions by: Shane Handley <https://github.com/shanehandley>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Now that the typings are included in the package, these fields are no longer needed

```ts
// Type definitions for unist-util-inspect 4.1
// Project: https://github.com/syntax-tree/unist-util-inspect/#readme
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

---

This should be moved to _package.json_ in the `contributors` field.

```ts
// Definitions by: Shane Handley <https://github.com/shanehandley>
```